src/main/java/uk/gov/ons/census/action/messaging/ActionFulfilmentReceiver.java

```diff
@@ -0,0 +1,95 @@
+package uk.gov.ons.census.action.messaging;
+
+import com.godaddy.logging.Logger;
+import com.godaddy.logging.LoggerFactory;
+import java.util.Map;
+import java.util.Optional;
+import java.util.UUID;
+import org.springframework.amqp.rabbit.core.RabbitTemplate;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.integration.annotation.MessageEndpoint;
+import org.springframework.integration.annotation.ServiceActivator;
+import org.springframework.transaction.annotation.Transactional;
+import uk.gov.ons.census.action.model.dto.PrintFileDto;
+import uk.gov.ons.census.action.model.dto.ResponseManagementEvent;
+import uk.gov.ons.census.action.model.dto.UacQidDTO;
+import uk.gov.ons.census.action.model.entity.ActionType;
+import uk.gov.ons.census.action.model.entity.Case;
+import uk.gov.ons.census.action.model.repository.CaseRepository;
+import uk.gov.ons.census.action.service.CaseService;
+
+@MessageEndpoint
+public class ActionFulfilmentReceiver {
+  private static final Logger log = LoggerFactory.getLogger(ActionFulfilmentReceiver.class);
+  private final RabbitTemplate rabbitTemplate;
+  private final CaseService caseService;
+  private final CaseRepository caseRepository;
+
+  @Value("${queueconfig.outbound-exchange}")
```

Comment:
Should these config values live in the AppConfig?

Reply:
this looks OK to me - the value is configurable in the application.yml